docs/user-guide/api-mediation-standalone.md

```diff
@@ -0,0 +1,18 @@
+# API Mediation Layer as a standalone component
+
+As a Zowe user, follow the procedure in this article to start the API Mediation Layer independently of other Zowe components. 
+By default, the Gateway, Zowe System Services, and Virtual Desktop start when
+ Zowe runs. To limit consumed resources when the Virtual Desktop or Zowe System
+ Services are not required, it is possible to specify which components start in the
+ context of Zowe. No change is required during the installation process to
+ support this setup.
+ 
+Once Zowe is installed, use the following procedure to limit which components start.
+
+**Follow these steps:**
+
+1. Open the file `<Zowe instance directory>/instance.env`.
+2. Find the property `LAUNCH_COMPONENT_GROUPS` and set `GATEWAY`
```

Comment:
This doesn't result in just the APIML starting. The GATEWAY component group includes the explorer UI and API components.
If you want to just run the APIML then you can set `ZWE_LAUNCH_COMPONENTS=discovery,gateway,api-catalog` (and caching-service if/when required). This was added for HA support and isn't in any of the automated testing, so probably worth manually checking it's still okay given the large number of refactors going on in that area

Reply:
The property `LAUNCH_COMPONENT_GROUPS`  set to  `GATEWAY` has been replaced with 
 property `ZWE_LAUNCH_COMPONENTS` and set to `discovery,gateway,api-catalog`